sunkit_image/radial.py

```diff
@@ -130,6 +130,69 @@ def _percentile_ranks_numpy_inplace(arr):
         raise NotImplementedError(msg)
     return ranking_func
 
+def _set_attenuation_coefficients(order, range_mean=None, range_std=None, cutoff=0):
```

Comment:
Can this be moved back to its original location?

Reply:
ok.